cmd/main.go

```diff
@@ -125,6 +126,37 @@ func lint(l *thriftcheck.Linter, filenames []string) (thriftcheck.Messages, erro
 	return l.LintFiles(filenames)
 }
 
+func getFilenames(inputs []string) ([]string, error) {
```

Comment:
I think I'd rename this to reflect that we're going from paths to filenames:

```suggestion
func expandPaths(paths []string) (filenames []string, error) {
```